index.html

```diff
@@ -135,7 +135,26 @@ <h4 class="skills">Skills</h4>
             <hr>
             
         </div>
-    </section> 
+    </section>    
+    <section class="contact">
+        <fieldset class="contact-form">
+            <div class="background-img">
+                <h2 class="contact-heading">Contact me</h2>
+                <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!</p>
+            <form action="" method="Post" class="details-form">
+                <input id="username" type="text" placeholder="Lorem Ipsum" required>
+                <input id="email" type="email" placeholder="Loremipsum@gmail.com" required>
+                <textarea id="textarea" placeholder="write your message..." required name="message" id="" cols="30" rows="10"></textarea>
```

Comment:
- [ ] Please make sure that the name input has the `maxlength="30"` attribute and the message input has the `maxlength="500"` attribute. :pray: